spec/fixtures/extensions/chrome-api/main.js

```diff
@@ -5,6 +5,17 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
 })
 
 const testMap = {
+  connect () {
+    let success = false
+    try {
+      chrome.runtime.connect(chrome.runtime.id)
+      chrome.runtime.connect(chrome.runtime.id, { name: 'content-script' })
+      chrome.runtime.connect({ name: 'content-script' })
+      success = true
+    } finally {
+      console.log(JSON.stringify(success))
```

Comment:
Is this console.log intentional or was it accidentally left in?

Reply:
I looked through the change again -- I didn't realize it was for the spec.  Looks like the console.log is intentional